src/registrar/config/settings.py

```diff
@@ -86,6 +92,10 @@
     # the "user" model! *\o/*
     "django.contrib.auth",
     # generic interface for Django models
+    "auditlog",
+    # library to simplify form templating
+    # it needs to be listed before django.contrib.contenttypes
```

Comment:
wait this comment confuses me. It sounds like this is referring to django.contrib.contenttypes but then it says it needs to be listed before django.contrib.contenttypes. Maybe this is meant to be a comment for auditlog?